pkg/utils/utils.go

```diff
@@ -51,6 +55,23 @@ func init() {
 	ClusterType = os.Getenv("CLUSTER_TYPE")
 }
 
+// IsKernelCmdLineParamSet This checks if the kernel cmd line is set properly.
+func IsKernelCmdLineParamSet(param string, chroot bool) (bool, error) {
```

Comment:
is `chroot` ever not false?

Reply:
I am just following utils.go : IsKernelLockdownMode() styling.
If the code chooses to call utils.go "Chroot(path string)" then we need it to be set true.